free-tunes/Controllers/TrackCollectionViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  TrackCollectionViewCell.swift
+//  Free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/03/13.
+//
+
+import UIKit
+
+class TrackCollectionViewCell: UICollectionViewCell {
+
+//  MARK: - Interface Builder Outlets
+    @IBOutlet weak var trackCover: UIImageView!
```

Comment:
Maybe make these variables private

Reply:
but that entails that the variables can't be accessed when setting up the cell in the  collection view

Reply:
You can use setters for those variables